compiler-rt/lib/asan/asan_interface_internal.h

```diff
@@ -202,6 +205,9 @@ extern "C" {
   SANITIZER_INTERFACE_ATTRIBUTE
   extern uptr *__asan_test_only_reported_buggy_pointer;
 
+  SANITIZER_INTERFACE_ATTRIBUTE
+  extern bool __asan_is_dormant;
```

Comment:
Other boolean-like options use `int` like `__asan_option_detect_stack_use_after_return`.

I personally don't mind requiring a new enough C ABI that we can use bool, but I'm not sure if there's some legacy issue that we should simply stick to `int`.